packages/core/get-own-property-keys.js

```diff
@@ -1,5 +1,5 @@
-module.exports = function getOwnPropertyKeys(obj) {
-  var keys = Object.getOwnPropertyNames(obj);
-  if (!Object.getOwnPropertySymbols) return keys;
-  return keys.concat(Object.getOwnPropertySymbols(obj));
-};
+'use strict';
+
+const { ownKeys: getOwnPropertyKeys } = Reflect;
```

Comment:
TIL `Reflect.ownKeys`

Awesome! Thank you.

Sound like we don't need this file any more. :) We can just use `Reflect.ownKeys` everywhere! Yay!

Reply:
Usage of `getOwnPropertyKeys()` has been changed for `Reflect.ownKeys()` directly.
The function `getOwnPropertyKeys()` is still available for `@stampit/core` but marked as `@deprecated`.